pages/doc.js

```diff
@@ -144,6 +169,23 @@ export default class Documentation extends Component {
       })
   }
 
+  loadFile = ({ section, subsection, file, parseHeadings }) => {
+    this.setState({ load: true })
+    let sect = sidebar[section]
+    let subsect = sect.files[subsection]
+    let subfolder = subsect && subsect.folder
+    let folderpath = file.folder || subfolder || sect.folder
+    let filepath = file.indexFile || file.files || file
```

Comment:
same here, previous lines are compilcated.

should it be actuallu `files[0]`?

Reply:
so, is `file.files` a bug? should it be `file.files[0]`?

Reply:
This function is used if the section (User Guide for example) does not contain its own root file, in which case the application suspects that the required file should be searched for in the attached file, and the latter may also be an object or a string.